elements/rhodium.md

```diff
@@ -6,10 +6,10 @@ title: Rhodium
 Symbol : Rh
 Atomic Number : 45
 
-Rhodium makes up only 0.00002 ppm of the earth's crust.
+Rhodium makes up only 0.00002 ppm of the earth's crust! 
 
-Fun Facts:
+Other Fun Facts:
 - ~87% of all rhodium produced is used for catalytic converters.
-- The price of an ounce of Rhodium (~$19,000) is over 10x the price of an ounce of gold (~$1,800).
+- The price of an ounce of Rhodium (\~$19,000) is over 10x the price of an ounce of gold (\~$1,800).
 - Rhodium-103 is the only stable isotope of rhodium.
 - Rhodium is a common metal used in jewelry plating, which some refer to as white gold.
```

Comment:
This is indeed a fun fact!

Reply:
The strikethrough should be solved also. Thanks!